File: Backend/TaskManagementSystem/TaskManagementSystem.API/DTOs/TaskDetailDto.cs
namespace TaskManagement.API.DTOs
{
    public class TaskDetailDto
    {
        public string UserName { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public DateTime? CreatedDate { get; set; }
        public DateTime? DueDate { get; set; }
        public DateTime? ClosedDate { get; set; }
        public string TaskStatus { get; set; }
        public string CreatedByName { get; set; }
        public string FeedBack { get; set; }
        public string UserTaskStatus { get; set; }
        public DateTime? AssignedDate { get; set; }
        public DateTime? UserTaskDueDate { get; set; }
        public DateTime? UserTaskClosedDate { get; set; }
    }
}

File: Frontend/TaskSystem/src/app/shared/models/task-detaildto.ts
import { Status } from "./status.enum";
export interface TaskDetailDto {
    userName: string;
    title: string;
    description: string;
    createdDate?: Date;
    dueDate?: Date;
    closedDate?: Date;
    taskStatus: Status;
    createdByName: string;
    feedBack: string;
    userTaskStatus: Status;
    assignedDate?: Date;
    userTaskDueDate?: Date;
    userTaskClosedDate?: Date;  
}
